users/greyhatmiddleman/pimoroni_trackball.c

```diff
@@ -0,0 +1,301 @@
+#include "pimoroni_trackball.h"
```

Comment:
This probably won't get merged until after develop is merged into master (end of august), because the pimoroni code has been moved to `/drivers/sensors/`, and would prefer that new copies not be added to the repo.  Instead, changes should be made to the core file.